tests/unit/subscribeToPropertiesTest.js

```diff
@@ -16,6 +16,23 @@ Onyx.init({
     registerStorageEventListener: () => {},
 });
 
+class ErrorBoundary extends React.Component {
```

Comment:
@marcaaron I ran into more cases where if there is a syntax error inside the selector function, then Jest isn't able to catch and report it.